### ISSUE TITLE
Help > Accessibility Options context menu should be hidden

### ISSUE THREAD

**Issue by mousetraps** [Microsoft/vscode#12932]
Testing #12519

We handle accessibility in our own way, so having the additional menu item from chromium is confusing. Not to mention, the resulting dialog itself is inaccessible (not resizeable, confusing icons, etc.) 
![image](https://cloud.githubusercontent.com/assets/762848/18929790/c4173098-857a-11e6-8c42-16edb8d1b929.png)

Related to https://github.com/Microsoft/vscode/issues/12928

**Comment by joaomoreno** [comment]
This is needed until the issue is resolved in Chromium/Electron. When it is, we'll simply remove the whole thing.

cc @alexandrudima @bpasero

**Comment by bpasero** [comment]
We should take this out for the stable release and only have it in insiders.

**Comment by bpasero** [comment]
Verify you see this entry in our insiders and out of sources on Windows but not in stable on Windows.

**Comment by joaomoreno** [comment]
@bpasero Using devtools without this box is practically impossible... not so sure if removing it is a good idea. We ask a lot of our users to user the devtools to find out more about problems.

**Comment by bpasero** [comment]
@joaomoreno in that case we can always ask them to use our insider build. which is a good advise anyways because often a bug does not show up in insiders build due to:
* the build not being patched by a bad extension
* the build not having the same extensions installed
* the build already including a fix

**Comment by mousetraps** [comment]
> Using devtools without this box is practically impossible...

Not sure I understand... Why is this? And what's the chromium issue that you alluded to? Can't seem to find any reference to it.

**Comment by joaomoreno** [comment]
A couple of weeks ago @alexandrudima  got a Surface Book. He was happy until he tried Code: it was super slow. After much investigation (with @bpasero & @dbaeumer) they found the problem surfaces on certain devices only and occurs when Chrome goes into some special accessibility mode, making things very slow.

Opening the dev tools in such a device (I believe any touchscreen device would trigger it) is a painfully slow experience.

Guys, was there an issue for this? I couldn't find it.

**Comment by mrmlnc** [comment]
@joaomoreno, maybe https://github.com/electron/electron/issues/7208 ?
